todo_app/app.py

```diff
@@ -1,11 +1,25 @@
-from flask import Flask
+from flask import Flask, redirect, url_for, request
+from flask import render_template
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template("index.html", items = items)
+
+@app.route('/addItem', methods=['POST'])
+
+def index_addItem():
```

Comment:
Convention in Python is for method names to be `lower_snake_case` (see PEP8: https://www.python.org/dev/peps/pep-0008/)
```suggestion
def add_item():
```